src/com/commandgeek/GeekSMP/listeners/InteractListener.java

```diff
@@ -125,4 +125,21 @@ public void onDestroyBlock(BlockBreakEvent event) {
             LockManager.unplace(event.getBlock());
         }
     }
+    
+    
+    //Will prevent Undead from bonemealing moss, preventing moss griefing
+    
+    @SuppressWarnings("deprecation")
+    public void onBoneMealCovert(PlayerInteractEvent e) {
+        ItemStack bonemeal = new ItemStack(Material.INK_SACK, 1, (short) 15);
```

Comment:
Shouldn't this be `BONE_MEAL` instead of `INK_SACK`?
And I don't think it needs the `, (short) 15`

Reply:
I have changed it from `INK_SAC` to `BONE_MEAL,` and removed the `, (short) 15`